ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis_object_storages.robot

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation     Collection of CLI tests to validate fetching models from different object storages
+...               in the scope of model serving stack for Large Language Models (LLM).
+...               These tests leverage on TGIS Standalone Serving Runtime
+Resource          ../../../../Resources/OCP.resource
+Resource          ../../../../Resources/CLI/ModelServing/llm.resource
+Resource          ../../../../Resources/CLI/Minio.resource
+Library           OpenShiftLibrary
+Suite Setup       Suite Setup
+Suite Teardown    RHOSi Teardown
+Test Tags         KServe
+
+
+*** Variables ***
+${MODEL_S3_DIR}=    flan-t5-small-hf
+${TEST_NS}=    tgis-storages
+${TGIS_RUNTIME_NAME}=    tgis-runtime
+
+  
+*** Test Cases ***
+Verify User Can Serve And Query A Model From Minio
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model
+    ...                using Kserve and TGIS runtime fetching models from a MinIO server
+    [Tags]    Tier1    RHOAIENG-3490
```

Comment:
isn't it missing the Sanity tag?

Reply:
on purpose cos I'm not sure if all the new tests for serving should fall in sanity which is growing a lot. We could review the quality gates tags all together for model serving